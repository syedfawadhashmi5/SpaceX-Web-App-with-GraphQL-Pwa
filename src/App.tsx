import React from "react";
import "./App.css";

import LaunchListContainer from "./Components/Launch";
import LaunchDetail from "./Components/LaunchDetails";
import { BrowserRouter as Router, Route } from "react-router-dom";

const App = () => {
  return (
    <div>
      <Router>
        <Route exact path="/">
          <div className="componentDiv">
            <LaunchListContainer />
            <LaunchDetail />
          </div>
        </Route>
      </Router>
    </div>
  );
};

export default App;
