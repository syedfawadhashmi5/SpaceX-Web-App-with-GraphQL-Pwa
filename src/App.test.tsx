import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('render App component', () => {

  const container = shallow(<App />);


  it('should render a div in App', () => {
    expect(container.find('div').length).toBeGreaterThanOrEqual(2);

    expect(container).toMatchSnapshot();

  });
  
})