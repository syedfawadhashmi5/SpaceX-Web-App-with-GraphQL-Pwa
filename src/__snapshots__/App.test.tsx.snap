// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App component should render a div in App 1`] = `
<div>
  <BrowserRouter>
    <Route
      exact={true}
      path="/"
    >
      <div
        className="componentDiv"
      >
        <LaunchListContainer />
        <LaunchInfo />
      </div>
    </Route>
  </BrowserRouter>
</div>
`;
